.github/workflows/update-semgrep-rules.yml

```diff
@@ -0,0 +1,59 @@
+name: Update Semgrep Rules
+on:
+  workflow_dispatch: {}
+  # TODO: Remove this trigger
+  push:
+    branches:
+      - "cron"
+  #TODO: at some point also set up an actual cron like
+  #  schedule:
+  #  - cron: "43 20 * * *"
+
+jobs:
+  update-semgrep-rules-submodule:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get JWT for semgrep-ci GitHub App
```

Comment:
why do we need that? What was wrong with the github-action default bot?

Reply:
The default one doesn't allow it to trigger downstream jobs - if the default token is used to open a PR, then checks won't run on that PR. That API might've just changed, though, I am confirming.

Reply:
See https://docs.github.com/en/enterprise-cloud@latest/actions/using-workflows/triggering-a-workflow#triggering-a-workflow-from-a-workflow for a more detailed explanation of why the default token is not sufficient.